### PR TITLE
remove debug echo

### PR DESCRIPTION
**Pull request by joemiller**

### CODE REVIEW

**Comment by joemiller** [comment]
image

File: scripts/push_packagecloud.sh [hunk 1]
```diff
@@ -15,5 +15,5 @@ if [ -z "$1" ] ; then
 fi
 
 for i in ${repo_versions[@]} ; do
-  echo package_cloud push "pantheon/$1/fedora/$i" pkgs/*.rpm
```
Comment (spheromak) [inline scripts/push_packagecloud.sh:18]: didn't we fix this a while back someplace else ?

**Comment by joemiller** [comment]
@greg-1-anderson feel free to merge this if you +1

File: scripts/push_packagecloud.sh (hunk 1 above)
Comment (joemiller) [inline scripts/push_packagecloud.sh:18]: in the rpmbuild-drush repo ;)

**Comment by spheromak** [comment]
:+1: :shipit: :rocket:

**Comment by greg-1-anderson** [comment]
I doubled my shame and pushed the equivalent change directly to `master`.  Does not affect production, and was what was intended.  Anyway, it did push to packagecloud without incident, once the `echo` was removed.

**Pull request closed without merging**